quic/s2n-quic-core/src/recovery/bbr/data_volume.rs

```diff
@@ -150,20 +153,42 @@ impl Model {
         self.inflight_hi = inflight_hi;
     }
 
-    /// Updates `inflight_lo` with the given `inflight_latest` if it exceeds
-    /// the current `inflight_lo` * `bbr::BETA`
-    pub fn update_lower_bound(&mut self, cwnd: u32, inflight_latest: u64) {
+    /// Updates `inflight_lo` according to if there is loss or ECN in the round
+    pub fn update_lower_bound(
+        &mut self,
+        cwnd: u32,
+        inflight_latest: u64,
+        loss_in_round: bool,
+        ecn_in_round: bool,
+        ecn_alpha: Ratio<u64>,
+    ) {
```

Comment:
Hmm, I'm not sure I like that better (though I agree the current version isn't great either). I don't really like pretending there isn't an `inflight_latest` or an `ecn_alpha` just because there was no loss or ECN in the round.